Kfilter/src/main/java/com/isaacudy/kfilter/KfilterView.kt

```diff
@@ -345,6 +353,26 @@ class KfilterView @JvmOverloads constructor(context: Context,
     }
     //endregion
 
+    fun test(isLeft: Boolean, x1: Float, x2: Float) {
```

Comment:
Please include a more descriptive name for this function, and make it private (as it;s not used outside this file)

Reply:
test -> checkFilterSwipeDirection
This method just check filter are applying in left direction or right